Exercicio3.rb

```diff
@@ -0,0 +1,22 @@
+#Exercicio3
+#Mior fator principal
+#Os fatores primos de 13195 são 5, 7, 13 e 29.
+
+#Qual é o maior fator primo do número 600851475143?
+
+def fatorPrimo(n)
+    i = 2
+    maior = 0
+    while (i <= n)
+      if (n % i == 0)
+        while (n % i == 0 )
```

Comment:
:nail_care: (nos code reviews, colocamos esse simbolo quando queremos fazer uma sugestão que é estética ou de gosto pessoal)

É muito mais fácil entender o código se as variaveis forem bem explícitas.

```suggestion
        while (number % index == 0 )
```